### PR TITLE
Mention Cloud Run and Container Instance

### PR DESCRIPTION
**Pull request by ahuang11**
I think we left them out before.

### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

File: docs/concepts/infrastructure.md [hunk 1]
```diff
@@ -33,6 +35,8 @@ Infrastructure is specific to the environments in which flows will run. Prefect
 - [`DockerContainer`](/api-ref/prefect/infrastructure/#prefect.infrastructure.docker.DockerContainer) runs flows in a Docker container.
 - [`KubernetesJob`](/api-ref/prefect/infrastructure/#prefect.infrastructure.kubernetes.KubernetesJob) runs flows in a Kubernetes Job.
 - [`ECSTask`](https://prefecthq.github.io/prefect-aws/ecs/) runs flows in an ECS Task.
+- [`Cloud Run`](https://prefecthq.github.io/prefect-gcp/cloud_run/) runs flows in a Cloud Run Job.
+- [`Container Instance`](https://prefecthq.github.io/prefect-azure/container_instance/) runs flows in a Container Instance.
```
Comment (madkinsz) [inline docs/concepts/infrastructure.md:39]: We should probably specify the cloud provider before each of these

Comment (ahuang11) [reply docs/concepts/infrastructure.md]: Added to the sentence description

**Review by tpdorsey — APPROVED** [review]
Looks good

**Review by anna-geller — APPROVED** [review]
nice

**Pull request merged**